portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,67 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```

Comment:
Generally I'm a bit uncomfortable with the use of external resources because the external site might be unavailable and hence there are additional ways that your site could stop working.

It's good to see that you have specified a specific integrity-checked version.